packages/consensus/src/logging.cairo

```diff
@@ -0,0 +1,17 @@
+#[cfg(feature: 'log_level_trace')]
```

Comment:
I can't find a way to import the logging constant to the desired package to use the constants. seems that the procedural macros package behaves differently because importing the constant from utils is fine though

Reply:
Maybe try to make package with macros a dependency of loggin package(which would be cairo only)?

Reply:
very weird even the compiler is happy after the feature is enabled, maybe this is cairo bug

```
error: Identifier not found.
 --> //raito/packages/consensus/src/codec.cairo:14:30
        if logging::logging::LOG_LEVEL_DEBUG { println!("{}: {}", format!("DEBUG"), format!("tx_encoded: {}", dest)) }
                             ^*************^

error: could not compile `consensus` due to previous error
```

![image](https://github.com/user-attachments/assets/2d008a72-1c8b-49bf-bc9a-82950e01836a)

Reply:
It should be `logging::log::LOG_LEVEL_DEBUG` not `logging::logging::LOG_LEVEL_DEBUG`:
![image](https://github.com/user-attachments/assets/107c8fb8-f11b-4bd9-b2ea-74888c89835e)

Reply:
@maciejka i previously rename the file then `logging::logging::LOG_LEVEL_DEBUG`. also `logging::log::LOG_LEVEL_DEBUG` also not work even though the language server recognize when the feature is enabled.

Reply:
You are right:
![image](https://github.com/user-attachments/assets/52a459f1-4668-47fc-8bd9-ece181364837)

Reply:
I asked in scarb group: https://t.me/c/1788692421/1887